### DISCUSSION TITLE
Any suggested tutorials or course to understand and use the boilerplate

### DISCUSSION THREAD

**Discussion by Rowee13** [brocoders/nestjs-boilerplate#1644] [Q&A]
Hello.

I am new to Nest JS and just completed a course a couple of weeks ago and I saw this boilerplate that I know can be a huge help. But I am having trouble customizing it to what my personal project requires like user profile (additional fields based on my design), etc. I somehow get some parts but still having a hard time.

Do you guys have recommended tutorials on YouTube or even a course to help me advance with my learning? I also expanding my knowledge in PostgreSQL and docker. I am aiming to be comfortable in using Nest JS and this boilerplate because I like how organized the code by following its requirements.

Thanks in advance for any help you guys can give and I appreciate your work here.

**Comment by Shchepotin** [comment]
@Rowee13 We plan to do a video course on boilerplate.

Right now, you can learn more about boilerplate from the documentation https://github.com/brocoders/nestjs-boilerplate/blob/main/docs/readme.md
